spacy/morphology.pyx

```diff
@@ -27,7 +28,7 @@ cdef class Morphology:
         self.strings = strings
 
     def __reduce__(self):
-        tags = set([self.get(self.strings[s]) for s in self.strings])
+        tags = set([self.get(key) for key, _ in self.strings.items()])
```

Comment:
Both versions of this look incorrect in general? It's supposed to be a list of morphology tags not a list of strings, which is already in the pickled data as `self.strings`?

It can stay buggy here (temporarily), but I don't understand what this particular change is intended to accomplish as part of this PR?

Reply:
Oh, this one should probably go to another PR like I mentioned above; will remove it from this one.

The intention was to avoid calculating the hash of the mophological feature string (which is what the `self.strings[s]` call was doing), since it's already present in the `StringStore` instance.

As for picking `tags`, it does seem unnecessary - we can just iterate over the `StringStore`'s values and pass them to the `Morphology.add()` method.

Reply:
Ah, I didn't enough at the details of `self.get`, which are weird.